bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-dragula.js",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
I think this happened when I rebased to your master, but I'm happy to change it back.